code/modules/uplink/uplink_item.dm

```diff
@@ -871,6 +878,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	item = /obj/item/weapon/c4
 	cost = 1
 
+/datum/uplink_item/device_tools/c4bag
```

Comment:
okay seriously if anybody knows how to fix this hit me up, otherwise im remaking the PR (which sucks because then all this feedback will be harder to find)